corehq/apps/locations/views.py

```diff
@@ -212,6 +212,9 @@ def _get_loc_types(self):
             'expand_from': loc_type.expand_from.pk if loc_type.expand_from else None,
             'expand_from_root': loc_type.expand_from_root,
             'expand_to': loc_type.expand_to.pk if loc_type.expand_to else None,
+            'include_without_expanding': (loc_type.include_without_expanding.pk
+                                          if loc_type.include_without_expanding else None),
```

Comment:
Oh, that's cool. I didn't know about that, and actually inadvertently overwrote that functionality for `expand_from` and `expand_to`.